src/main/scala/no/ndla/searchapi/service/search/TaxonomyFiltering.scala

```diff
@@ -68,11 +67,9 @@ trait TaxonomyFiltering {
       Some(
         boolQuery().should(
           taxonomyFilters.map(
-            filterName =>
+            filterId =>
               nestedQuery("contexts.filters").query(
-                boolQuery().should(
-                  ISO639.languagePriority.map(l => termQuery(s"contexts.filters.name.$l.raw", filterName))
-                )
+                termQuery(s"contexts.filters.filterId", filterId)
```

Comment:
Har sikkert fint lite å si, men for ytelse så tror jeg det er bedre med en nestedQuery med en boolQuery i istedet for å ha flere nested queries.

Reply:
Men er det ikkje berre en nested query her?

Reply:
Det var visst en rett over her.

Reply:
Det blir vel en `nestedQuery` pr `filterId` når den mapper over her.
Hvis vi gjør feks sånn her, så vil vi få bare 1 nestedQuery med boolQuery istedet:
```
Some(
  nestedQuery("contexts.filters").query(
    boolQuery().should(
      taxonomyFilters.map(filterId => termQuery(s"contexts.filters.filterId", filterId))
    )
  ))
```

Reply:
Alle filtermetodene her returnerer en Option[BoolQuery] så då må vi vel legge det ytters. En bool med en nested som igjen har bool.

Reply:
Det er ikke noe problem om noen returnerer `Option[BoolQuery]` og andre `Option[NestedQuery]` så vidt jeg kan se?
begge passer vel inn i `Query` traitet som `getSearchFilters` returnerer